docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -33,13 +33,9 @@ refs:
 
 Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
 
-For more information on recording rules in Prometheus, refer to [Defining recording rules in Prometheus](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/).
-
-Recording rules are run as instant rules, which means that they run every 10s. To overwrite this configuration, update the min_interval in your custom configuration file.
+Grafana-Managed Recording Rules offer the same Prometheus-like semantics, but allow you to query any Grafana datasource supported by alerting. In addition to the above, you can use this to import and map data from other datasources into Prometheus.
 
-[min_interval](ref:configure-grafana) sets the minimum interval to enforce between rule evaluations. The default value is 10s which equals the scheduler interval. Rules will be adjusted if they are less than this value or if they are not multiple of the scheduler interval (10s). Higher values can help with resource management as fewer evaluations are scheduled over time.
```

Comment:
Only applies to GMRR, so moved it to that section.

Also this is extremely advanced, so i moved it to the bottom. I have never seen a use case for configuring this.